gradle.properties

```diff
@@ -31,7 +31,7 @@ javaxJaxbCoreVersion=2.3.0.1
 javaxJaxbImplVersion=2.3.2
 
 jaxRsVersion=2.1.6
-jerseyVersion=2.30.1
+jerseyVersion=2.31
```

Comment:
2.32 was released in September: https://mvnrepository.com/artifact/org.glassfish.jersey.core/jersey-server/2.32

Reply:
+1. followup PR (https://github.com/apple/servicetalk/pull/1207) to update in increments